package.json

```diff
@@ -2,7 +2,15 @@
   "name": "icd10",
   "version": "0.0.1",
   "description": "Package that gole is to gelp working with ICD Classificantion created by WHO",
+  "main": "dist/src/icd.js",
+  "types": "dist/src/icd.d.ts",
   "scripts": {
+    "build": "tsc",
+    "test": "node ./dist/src/test.js",
+    "generateXSD": "perl generators/dtd2xsd.pl data/icdClaML2016ens/ClaML.dtd >> ClaML.xsd",
```

Comment:
what is dtd2xsd?

Reply:
It's converter created by W3C, for now we don't use it so I will remove it from `package.json`